src/Aquarius.ONE.ClientSDK/Common/Logbook/LogbookCache.cs

```diff
@@ -94,7 +94,7 @@ public async Task<bool> LoadLogbooksAsync(string operationId = "")
         public async Task<bool> LoadEntriesByLogbookAsync(string logbookId, DateTime startDate, DateTime endDate)
         {
             if (!Logbooks.ContainsKey(logbookId))
-                return ErrorResponse(UnloadedException(logbookId), false);
+                return ErrorResponse(CacheExceptions.UnloadedException("Logbook", logbookId), false);
```

Comment:
Was this change to share error handling across areas of the SDK?

Reply:
yes, it is the class to share error handling across the all Cache classes